checkstyle/licenses/checkstyle-LICENSE.txt

```diff
@@ -0,0 +1,503 @@
+                  GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
@tomcallahan could you take a look at this? We depend on checkstyle in the build and this extension is part of the build.